projects/project-player-library/src/lib/pages/preview-details-page/preview-details-page.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component } from '@angular/core';
+import { projectDetailsData } from '../details-page/project-details.component.spec.data';
+import { actions } from '../../constants/actionConstants';
+import { RoutingService } from '../../services/routing/routing.service';
+import { ActivatedRoute } from '@angular/router';
+import { DbService } from '../../services/db/db.service';
+import { ToastService } from '../../services/toast/toast.service';
+import { ApiService } from '../../services/api/api.service';
+
+@Component({
+  selector: 'lib-preview-details-page',
+  templateUrl: './preview-details-page.component.html',
+  styleUrl: './preview-details-page.component.css'
+})
+export class PreviewDetailsPageComponent {
+  projectDetails:any ;
+  actionsList = [];
+  displayedTasks:any;
+  remainingTasks = [];
+  startImprovement: boolean = true;
+  solutionId:any;
+  constructor(private routerService:RoutingService,private activatedRoute:ActivatedRoute,private db:DbService,private toasterService:ToastService,private apiService:ApiService){
+    activatedRoute.params.subscribe(param=>{
+     this.solutionId = param['id']
+     this.getProjectTemplate()
+      this.getData(this.solutionId)
+    })
+  }
+  ngOnInit(): void {
+  }
+
+  getProjectTemplate(){
+    const configForSolutionId = {
```

Comment:
change configForSolutionId to configForTemplateDetails

Reply:
done